### PR TITLE
speed up searching in augeas key cache

### PR DESCRIPTION
**Pull request by jreidinger**
I do not bump version as this small change do not deserve it itself as speed up is quite small ( around 2% )

### CODE REVIEW

File: lib/cfa/augeas_parser.rb [hunk 1]
```diff
@@ -283,12 +283,7 @@ def initialize(aug)
 
   # returns list of keys available on given prefix
   def keys_for_prefix(prefix)
-    cut = prefix.length > STORE_LEN ? STORE_LEN_1 : STORE_LEN
```
Comment (jreidinger) [inline lib/cfa/augeas_parser.rb]: you are right

**Comment by mvidner** [comment]
The changed code is covered: https://coveralls.io/builds/8315267/source?filename=lib%2Fcfa%2Faugeas_parser.rb#L274

> I do not bump version as this small change do not deserve it itself as speed up is quite small ( around 2% )

But since you change caching code there is a risk of bugs. Do you really want to delay them until you no longer remember the context? I would go for releasing the change.

**Comment by jreidinger** [comment]
@mvidner well, it is some work to release gem and I do not like to release everyday. I think also other gems authors do not release gems after each commit ( and each commit can introduce bug ).

**Comment by jreidinger** [comment]
@mvidner thanks

**Pull request merged**